From 1ca09a1a8f3dd07e2805855a1f1a67a54e8d8adb Mon Sep 17 00:00:00 2001
From: Maxime Hadjinlian <maxime.hadjinlian@gmail.com>
Date: Wed, 15 Jan 2014 00:20:08 +0100
Subject: [PATCH 2/3] Fix installation error

ln will complain that the link already exists and will error.
Adding the -f flag, force ln to recreate the link without failing the
build.

Signed-off-by: Maxime Hadjinlian <maxime.hadjinlian@gmail.com>
---
 Makefile.am     | 2 +-
 src/Makefile.am | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index b5b50be..ddf1d5f 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -54,7 +54,7 @@ install-data-hook:
 	cd $(DESTDIR)$(man1dir); \
 	inst1=`echo enca | sed '$(transform)'`.1; \
 	inst2=`echo enconv | sed '$(transform)'`.1; \
-	$(LN_S) $$inst1 $$inst2
+	$(LN_S) -f $$inst1 $$inst2
 
 uninstall-hook:
 	inst=`echo enconv | sed '$(transform)'`.1; \
diff --git a/src/Makefile.am b/src/Makefile.am
index 2f01a12..3885137 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -47,7 +47,7 @@ install-exec-hook:
 	cd $(DESTDIR)$(bindir); \
 	inst1=`echo enca | sed '$(transform)'`; \
 	inst2=`echo enconv | sed '$(transform)'`; \
-	$(LN_S) $$inst1$(EXEEXT) $$inst2$(EXEEXT)
+	$(LN_S) -f $$inst1$(EXEEXT) $$inst2$(EXEEXT)
 
 uninstall-hook:
 	inst=`echo enconv | sed '$(transform)'`; \
-- 
1.8.5.2

